test/web-platform-tests/to-run.yaml

```diff
@@ -192,11 +226,49 @@ DIR: dom/events
 Event-dispatch-click.tentative.html: [fail, Test is wrong, https://github.com/web-platform-tests/wpt/issues/27819]
 Event-timestamp-high-resolution.html: [fail, Not implemented]
 Event-timestamp-high-resolution.https.html: [fail, Not implemented]
+EventListener-handleEvent-cross-realm.html: [fail, unknown iframe issue]
 EventListener-incumbent-global-1.sub.html: [timeout, Multi-globals]
 EventListener-incumbent-global-2.sub.html: [timeout, Multi-globals]
 EventListener-invoke-legacy.html: [timeout, Animation stuff not implemented]
 EventTarget-dispatchEvent.html: [fail, We don't support every event interface yet]
 event-global-extra.window.html: [fail, We're supposed to check the event listener's global (not the Event's global)]
+event-global-is-still-set-when-coercing-beforeunload-result.html: [fail-slow, unknown iframe issue]
+event-global-is-still-set-when-reporting-exception-onerror.html: [fail, unknown iframe issue]
+event-global-set-before-handleEvent-lookup.window.html: [fail, unknown iframe issue]
+mouse-event-retarget.html: [fail, Requires a layout engine]
+passive-by-default.html:
```

Comment:
rather than just `fail` the whole file, i enumerated the failures, as the `touchend` variants do all pass under jsdom. i haven't looked into the specifics though, so maybe it'd be better to just `fail` the whole file, due to the lack of Pointer events support?

Reply:
Yeah, I think it'd be best to fail this entire file, as it's about a spec change we have not worked on: https://github.com/whatwg/dom/commit/b29449709a7b797cdb9721600bb552f8f140e64a